frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/contextmenu/CheckOutBranchAction.java

```diff
@@ -32,7 +28,7 @@
  *  <li>{@link DataKeys#KEY_SELECTED_VCS_REPOSITORY}</li>
  * </ul>
  */
-public class CheckOutBranchAction extends AnAction implements DumbAware {
+public class CheckOutBranchAction extends DumbAwareAction {
```

Comment:
Maybe while you're here, rename to `CheckOutSelectedBranchAction` for clarity & consistency (even though we'll obviously never have `CheckOutCurrentBranchAction`)

Reply:
Btw https://github.com/VirtusLab/git-machete-intellij-plugin/issues/266 rel

Reply:
taking!